Assets/MixedRealityToolkit/Attributes/MixedRealityControllerAttribute.cs

```diff
@@ -60,9 +61,18 @@ public class MixedRealityControllerAttribute : Attribute
         /// <summary>
         /// Convenience function for retrieving the attribute given a certain class type.
         /// </summary>
+        /// <remarks>
+        /// This function is only available in a UnityEditor context.
```

Comment:
I don't see a check for editor in this file. Did I miss something?

Reply:
Nah, I think I had merged my change with some other commit I had (i.e. I had a cherry pick which unity editor-ed this entire thing out). I opted to go with this approach instead since there actually was an alternative here.